### PR TITLE
Do not render until all texture resources are available

### LINKED ISSUE #814: Postpone scene rendering until all resources are ready
Currently we start rendering a scene if texture resources are not ready and are in process of being downloaded. And the current state of the scene is to substitute dummy textures, which get replaced once actual resources are fetched.

Instead of the current approach of rendering dummy resources, we would like to postpone the scene rendering until all scene's resources are ready. (This should also include a tile's raster texture resource, in that case).

### PR DESCRIPTION
**Pull request by tallytalwar**
Fixes #814

### CODE REVIEW

**Comment by hjanetzek** [comment]
Does this trigger a render request once all textures are available?

**Comment by tallytalwar** [comment]
Hmm.. good point. I do not think so. Will look.

**Comment by tallytalwar** [comment]
@hjanetzek Updated.

File: core/src/scene/sceneLoader.cpp [hunk 1]
```diff
@@ -596,7 +598,12 @@ std::shared_ptr<Texture> SceneLoader::fetchTexture(const std::string& name, cons
                     }
 
                     updateSpriteNodes(name, texture, scene);
-                    scene->resourceLoad--;
+                    using namespace std::chrono_literals;
+                    std::this_thread::sleep_for(10s);
```
Comment (tallytalwar) [inline core/src/scene/sceneLoader.cpp]: Ohh yes, updated.

**Comment by tallytalwar** [comment]
Rebased and Updated. Ultra small code change here!

**Pull request merged**